contrib/inventory/vmware.py

```diff
@@ -330,6 +339,13 @@ def get_inventory(self, meta_hostvars=True):
                 if prefix_filter:
                     if vm.name.startswith( prefix_filter ):
                         continue
+                if instance_filters:
+                    included = False
+                    for instance_filter in instance_filters:
+                        if vm.name.startswith( instance_filter ):
```

Comment:
Since we already have a prefix_filter, it may be better to use instance_filter as a general search term:
```
for instance_filter in instance_filters:
    if instance_filter in vm.name:
```